RELEASE.rst

```diff
@@ -0,0 +1,24 @@
+Release Procedure for pyvo
+==========================
+
+These steps are intended to help guide a developer into
+making a new release.  For these instructions, version
+is to be replaced by the version number of the release.
+
+1. Edit setup.cfg and remove .dev from the version number
+
+2. Edit CHANGES.rst to change unreleased to the date of the release.
+
+3. Commit and push
+
+4. python setup.py sdist (this makes a .tar.gz of the package in dist)
+
+5. twine upload sdist/* (this uploads the output of the previous step to pypi)
+
+6. git tag -a version -m "releasing new version version" (this makes a release tag)
```

Comment:
It doesn't matter here as there are no wheels/etc involved, but ideally the tagging is happening before pushing to pypi. The only irreversible step in the  process is the pypi upload, so that should happen the latest (sometimes there can be extra testing for the tags that reveal issues, so then that tag can be updated before the final release is done and announced.)

Reply:
Okay sounds good reordered those steps